src/Orleans.Http.MediaTypes.Protobuf/PipeReaderStream.cs

```diff
@@ -0,0 +1,240 @@
+namespace Orleans.Http.MediaTypes.Protobuf
+{
+    using System;
+    using System.Buffers;
+    using System.IO;
+    using System.IO.Pipelines;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    internal class PipeStream : Stream
```

Comment:
Why do you need this? Just call AsStream on the PipeWriter/PipeReader

Reply:
Indeed. I don't remember now why I had an issue. Just ran the tests and it worked...